metrics/core/src/main/java/org/hyperledger/besu/metrics/SyncDurationMetrics.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright contributors to Hyperledger Besu.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.metrics;
+
+import org.hyperledger.besu.metrics.noop.NoOpMetricsSystem;
+import org.hyperledger.besu.plugin.services.MetricsSystem;
+import org.hyperledger.besu.plugin.services.metrics.LabelledMetric;
+import org.hyperledger.besu.plugin.services.metrics.OperationTimer;
+
+import java.util.HashMap;
+
+/**
+ * This class manages the synchronization duration metrics for the Hyperledger Besu project. It
+ * provides methods to start and stop timers for various synchronization phases.
+ */
+public class SyncDurationMetrics {
+
+  /** A {@link SyncDurationMetrics} instance that does not record any metrics. */
+  public static final SyncDurationMetrics NO_METRICS_SYNC_DURATION_METRICS =
+      new SyncDurationMetrics(new NoOpMetricsSystem());
+
+  private final LabelledMetric<OperationTimer> timer;
+
+  private final HashMap<String, OperationTimer.TimingContext> timers = new HashMap<>();
+
+  /**
+   * Creates a new {@link SyncDurationMetrics} instance.
+   *
+   * @param metricsSystem The {@link MetricsSystem} to use to record metrics.
+   */
+  public SyncDurationMetrics(final MetricsSystem metricsSystem) {
+    timer =
+        metricsSystem.createSimpleLabelledTimer(
+            BesuMetricCategory.SYNCHRONIZER, "sync_duration", "Time taken to sync", "name");
```

Comment:
```suggestion
            BesuMetricCategory.SYNCHRONIZER, "total_sync_duration", "Total time taken to sync", "name");
```

Would be nice to state in the description (or at least the comment) exactly what this is measuring in terms of the stages, e.g. chain download, heal, backwards sync, etc

Reply:
What is the purpose of the "name" string here?

Reply:
hmmm...is "sync_duration" actually a label for a group of metrics?

Reply:
actually sync_duration is the actual name of the metric, which then has multiple children, which we identify with a label.